### PR TITLE
Create integration test codes

### PR DESCRIPTION
**Pull request by astrod**
## Description
Add test code for integration test. 
- Self-check > Create 
- Project >
    - Identification > Request Review, Confirm
    - Packaging > Request Review, Start Review
- OSS List > Create

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Refactoring, Maintenance
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by astrod** [comment]
Please add sign-off in commit
```
Commit sha: 5c2170e, Author: LeeJaeHyuk, Committer: LeeJaeHyuk; The sign-off is missing.
Commit sha: af5fd7c, Author: mingukang, Committer: mingukang; The sign-off is missing.
Commit sha: 3fe8bf4, Author: LeeJaeHyuk, Committer: LeeJaeHyuk; The sign-off is missing.
Commit sha: 52a82a9, Author: LeeJaeHyuk, Committer: LeeJaeHyuk; The sign-off is missing.
```

FYI @Lee-JaeHyuk , @mingukang-kr . See https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

**Comment by riyenas0925** [comment]
If you have write permission, you can add sign-off by using the command below.

https://github.com/fosslight/fosslight/pull/254/checks?check_run_id=4139167016


DCO
You have 4 commits incorrectly signed off. To fix, first ensure you have a local copy of your branch by checking out the pull request locally via command line. Next, head to your local branch and run:
```
git rebase HEAD~12 --signoff
```

Now your commits will have your sign off. Next run
```
git push --force-with-lease origin develop
```
Commit sha: 5c2170e, Author: LeeJaeHyuk, Committer: LeeJaeHyuk; The sign-off is missing.
Commit sha: af5fd7c, Author: mingukang, Committer: mingukang; The sign-off is missing.
Commit sha: 3fe8bf4, Author: LeeJaeHyuk, Committer: LeeJaeHyuk; The sign-off is missing.
Commit sha: 52a82a9, Author: LeeJaeHyuk, Committer: LeeJaeHyuk; The sign-off is missing.

View more details on DCO

**Comment by soimkim** [comment]
Dear @astrod ,
The details written in the Description were updated with the contents of the corrections in the PR.

**Review by riyenas0925 — APPROVED** [review]

**Pull request merged**
